cfme/automate/dialog_element.py

```diff
@@ -120,12 +123,20 @@ def reorder_elements(self, add_element, second_element, element_data):
         # Add a new element and then interchange position (BZ-1238721)
         if add_element:
             view.plus_btn.item_select("Add a new Element to this Box")
-            view.fill(second_element)
+            view.fill(second_element.get('element_information'))
+            if second_element.get('options') is not None:
+                view.fill(second_element.get('options'))
             view.element_tree.click_path(*self.tree_path[1:])
-        self.browser.drag_and_drop(self.element_loc(element_data.get("ele_label")),
-            self.element_loc(second_element.get("ele_label")))
+        dragged_el = element_data.get('element_information').get("ele_label")
```

Comment:
**Question**

Large PR, I may find an answer as I keep reading. There are drag_and_drop methods for the DialogElements Widget and DragAndDropElements View in WT_manageiq below. Can you use them here, instead of browser calls and elements outside of WT?

Reply:
Done